parquet/src/test/java/org/apache/iceberg/parquet/TestParquet.java

```diff
@@ -219,6 +221,50 @@ public void testTwoLevelList() throws IOException {
     assertThat(recordRead.get("topbytes")).isEqualTo(expectedBinary);
   }
 
+  @Test
+  public void testColumnStatisticsEnabled() throws Exception {
+    Schema schema =
+        new Schema(
+            optional(1, "int_field", IntegerType.get()),
+            optional(2, "string_field", Types.StringType.get()));
+
+    File file = createTempFile(temp);
+
+    List<GenericData.Record> records = Lists.newArrayListWithCapacity(5);
+    org.apache.avro.Schema avroSchema = AvroSchemaUtil.convert(schema.asStruct());
+    for (int i = 1; i <= 5; i++) {
+      GenericData.Record record = new GenericData.Record(avroSchema);
+      record.put("int_field", i);
+      record.put("string_field", "test");
+      records.add(record);
+    }
+
+    write(
+        file,
+        schema,
+        ImmutableMap.<String, String>builder()
+            .put(PARQUET_COLUMN_STATS_ENABLED_PREFIX + "int_field", "true")
```

Comment:
can we also test the default behavior ?

Reply:
I think we can test the default config after https://github.com/apache/parquet-java/pull/3189 is released. Shall we log an issue so we will remember?

Reply:
Yeah, the default behavior is true for now. If it is configured to false, it will not work as Huaxin mentioned. I can create a PR to add test coverage.

Reply:
We should add this to the Iceberg documentation as well. So users will be aware of the parquet issue, and don't expect it to work.

Reply:
Mentioned in the document.